metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-rest-env.xml

```diff
@@ -90,17 +90,11 @@
     </property>
     <property>
         <name>metron_temp_grok_path</name>
-        <description>Temporary local file path where grok patterns are written during testing</description>
-        <value>./patterns/temp</value>
+        <description>Temporary HDFS file path where grok patterns are written during testing</description>
+        <value>/apps/metron/patterns/tmp</value>
         <display-name>Metron temp grok path</display-name>
```

Comment:
This should not be hard coded, the user can change the metron hdfs root

Reply:
Done